### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by stevecj** [cdmoro/bootstrap-vue-3#213] [General]
After success with my initial experiment, my company is strongly considering using `bootstrap-vue-3` on our project. In that case, it might be in our interest to have me contribute some effort to the progress & stability of `bootstrap-vue-3`.

I am an experienced developer but very new to Vue and to the newer JS/ES toolsets, etc.

Perhaps, there are some simple sub-tasks that folks like me can take care of to get our feet wet and free up other developers' time for the core effort in the short term?

**Comment by cdmoro** [comment]
Hello! I'm very happy your company decided to use this library.
Answering your question, you could contribute to this project in these main areas:

- Create/maintain components
- Create missing directives
- Resolve issues
- Test components
- Maintain documentation
- Take a look at our parity list in order to complete missing components
- Prioritize what components we should create first
- Reach out to the Discussions section, there's always something to care of

As you can see, there's a lot to do... welcome to this early (but lovely) project! 😄

**Reply by stevecj** [reply]
Thanks. It turns out we have actually decided to use BV with Vue 2 for now, but counting on BV3 as an upgrade path to get to Vue 3 + Bootstrap 5 in the very near future, so I'm definitely still invested in the success of BV3 and plan to contribute.
